src/components/Book.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+const Book = ({ category, title }) => (
+  <tr>
```

Comment:
- [ ] It represents single row of books with id, title, and category [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Book/RepresentARowInBooksTable.png) (You are missing the ID, please add it to comply with the project requirements)

Reply:
Since the book component accepts a book object as props, id is not missing and being displayed anymore.